support/releaseinfo/static_provider.go

```diff
@@ -29,6 +29,14 @@ func (p *StaticProviderDecorator) Lookup(ctx context.Context, image string, pull
 	if err != nil {
 		return nil, err
 	}
+
+	releaseImage.Spec.Tags = append(releaseImage.Spec.Tags, imageapi.TagReference{
```

Comment:
why just not add an image [here](https://github.com/openshift/hypershift/blob/main/control-plane-operator/main.go#L357-L364)?

Reply:
at that point we don't know what the release image is yet.

Reply:
it is available to the process: https://github.com/openshift/hypershift/blob/e6407b07158d3abf317f02021751c55e44d2166b/control-plane-operator/main.go#L417

Reply:
nice, do you know since when we set this env variable? I am asking if we want backport this.

Reply:
Looks like long enough ago that it should be in all supported versions 
https://github.com/openshift/hypershift/pull/1023

Reply:
updated